docker-compose.yml

```diff
@@ -7,13 +7,17 @@ services:
     volumes:
       - './:/srv/pim'
     working_dir: '/srv/pim'
+    networks:
```

Comment:
There is no needs to declare a network on CE std, using the default one created by compose is enough.

Reply:
It's declared in EE-standard and this reduce the amount of diff when diffing between dev and standard. And explicit is good ;)

Reply:
OK then.